.github/workflows/auto-merge.yml

```diff
@@ -9,8 +9,27 @@ permissions:
 jobs:
   dependabot:
     runs-on: ubuntu-latest
-    if: ${{ github.actor == 'dependabot[bot]' && contains(github.event.pull_request.labels.*.name, 'python') }}
+    if: ${{ github.actor == 'dependabot[bot]' }}
     steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
```

Comment:
Hmm, not sure if this counts as an external action or not, so should be pinned via hash or not.

dependabot is a github thing, so I think that counts as internal?